### ISSUE TITLE
Once you get an error message you can't get rid of it

### ISSUE THREAD

**Issue by jdunkerley** [enso-org/enso#6976]
An error caused by a visualization timeout was shown (great to see) but it is impossible to get rid of this message so remains there until you reload Enso.

Need a dismiss button (or to dismiss after a period of time).

**Comment by wdanilo** [comment]
Ok, this is surprising. Im wondering if it was correctly working when the code was Q/Ad. I can't believe this could not be seen during Q/A. Is this a bug that was not visible earlier?

**Comment by sylwiabr** [comment]
![image](https://github.com/enso-org/enso/assets/12892578/68e2c5e5-6ec8-44f8-8bd4-4cfef6a52a5c)
here is the screen - some visualization updates failed on start of the project - IDE shows the message and I can't get rid of it.

**Comment by farmaazon** [comment]
> Ok, this is surprising. Im wondering if it was correctly working when the code was Q/Ad. I can't believe this could not be seen during Q/A. Is this a bug that was not visible earlier?

I tested the branch, and honestly I've seen the message. But I assumed the timeouts on visualization should happen very rarely (and undisposable message is still better than no message). For example, I needed to do very serious network throttling in the browser to see those timeouts, they never happen to me during everyday work.

This is a general problem of the old "status bar", enhanced by the fact the component no longer looks like status bar since it was moved from left-bottom corner of the scene. The proper implementation for the messages for the user is tracked in #5201. 

For now, we can revert the change, or add some mechanism of hiding - maybe use the co-existing "pop-up message" component, used when, for example, after failing to change project name.
